ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -175,3 +175,43 @@ def test_torch_leaky_relu(
         input=np.asarray(x, dtype=input_dtype),
         negative_slope=alpha,
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        )
+    ),
+    as_variable=st.booleans(),
```

Comment:
please remove `as_variable`, `with_out` and `native_array` here, they are generated internally through the `@handle_cmd_line_arg` decorator

but remain to **keep** them in the argument lists in `test_torch_tanh`